ci-operator/step-registry/sdn/conf/single-manifest/sdn-conf-single-manifest-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: sdn-conf-single-manifest
+  from: base
+  commands: sdn-conf-single-manifest-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    The sdn single manifest step adds the configuration for sdn single tenant networking to set of manifests for installation
```

Comment:
nit: "sdn" -> "SDN" and add a trailing period.

Reply:
OK